hack/dev

```diff
@@ -1,12 +1,11 @@
 #!/usr/bin/env bash
 
-set -e -u
+# NOTE: this script is kept for legacy reasons - you should just be able to
+# directly run the binaries produced by hack/build.
```

Comment:
it's really not legacy, it lets you run other shit with all the appropriate envs set... running `go test` is a huge pain without this.